java/src/jmri/server/json/util/JsonUtilHttpService.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Arrays;
 import java.util.Enumeration;
 import java.util.Locale;
-import javax.annotation.Nullable;
+import javax.annotation.CheckForNull;
```

Comment:
This must be Nullable per https://static.javadoc.io/com.google.code.findbugs/jsr305/3.0.2/javax/annotation/Nullable.html and http://findbugs.sourceforge.net/manual/annotations.html since this is overriding a Nonnull annotation, and is called out for use in that manner by those documents.

Reply:
The linked docs are also consistent with using `@CheckForNull` here:  "Use CheckForNull to indicate that the element value should always be checked for a null value."